docs/schemas/README.md

```
@@ -0,0 +1,201 @@
+  claude-pilot list --output $format | ajv validate -s docs/schemas/session-list.json
+done
+```
+
+## Support
+
+For questions about schema usage or to report schema issues:
+
+1. Check existing CLI output matches schema definitions
+2. Verify schema version matches CLI version compatibility
+3. Review migration documentation for version changes
+4. Open issue with schema validation details and CLI version info
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)